eng/targets/Helix.Common.props

```diff
@@ -69,9 +69,9 @@
     </When>
     <Otherwise>
       <PropertyGroup>
-        <IsArm64HelixQueue>$(HelixTargetQueue.Contains('Arm64'))</IsArm64HelixQueue>
-        <IsWindowsHelixQueue>$(HelixTargetQueue.Contains('Windows'))</IsWindowsHelixQueue>
-        <IsMacHelixQueue>$(HelixTargetQueue.Contains('OSX'))</IsMacHelixQueue>
+        <IsArm64HelixQueue>$(HelixTargetQueue.ToUpperInvariant().Contains('ARM64'))</IsArm64HelixQueue>
```

Comment:
this should be "or ARMARCH"... or just ARM since we're getting rid of the last ARM32s

Reply:
No, `call runtests.cmd AuthSamples.FunctionalTests.dll 8.0.0-ci windows.11.arm64.open arm64 false 00:45:00 false` is right at the top of the latest aspnetcore-helix-matrix run for this PR. The Win11 arm64 tests only run in that pipeline and I kicked it off manually.

This line works fine for the current Win11 build as well as `Debian.11.Arm64.Open` though the underlying image (`Ubuntu.1804.Armarch.Open`) wouldn't match. I think we're fine until / unless we update that friendly name or move to a Win11 queue using `armarch`.

Reply:
However, I did have a bug in a previous iteration which caused `runtests.sh` to be used. That's why I added the `ToUpperInvariant()` calls.